src/ncp/spinel.c

```diff
@@ -128,6 +128,8 @@ typedef struct {
     va_list obj;
 } va_list_obj;
 
+#define SPINEL_MAX_PACK_LENGTH         32767
```

Comment:
Why not use `SPINEL_FRAME_MAX_SIZE` instead here?

Reply:
Because that is the limit for valid spinel frames, this is the limit for the spinel parser functions. They are related, but they are different values. For example, you could have a buffer size of 2kb. Totally valid, if not overkill. However, if that limit were imposed here, then a 2kb buffer would not work. Better to have protocol-mandated frame-size limits imposed elsewhere.

Reply:
Sounds good.